src/basic/path-util.c

```diff
@@ -221,6 +221,33 @@ int path_make_relative_parent(const char *from_child, const char *to, char **ret
         return path_make_relative(from, to, ret);
 }
 
+int path_make_relative_cwd(const char *p, char **ret) {
+        char *c;
+        int r;
+
+        assert(p);
+        assert(ret);
+
+        if (!path_is_absolute(p))
```

Comment:
if you invert the test you could switch the if/else blocks an save ONE! character!